### ISSUE TITLE
Error in installation of package

### ISSUE THREAD

**Issue by rmcnellis** [EDIorg/EMLassemblyline#112]
I'm getting an error when trying to install the package from github

```
> remotes::install_github("EDIorg/EMLassemblyline")
Downloading GitHub repo EDIorg/EMLassemblyline@HEAD
Downloading GitHub repo EDIorg/taxonomyCleanr@HEAD
Error: Failed to install 'EMLassemblyline' from GitHub:
  Failed to install 'taxonomyCleanr' from GitHub:
  HTTP error 404.
  Not Found

  Did you spell the repo owner (`EDIorg`) and repo name (`EDIutils`) correctly?
  - If spelling is correct, check that you have the required permissions to access the repo.
```

**Comment by yvanlebras** [comment]
Hi Risa,

Maybe "just" a transient issue with your internet connexion no ? Can you retry ?

**Comment by clnsmth** [comment]
Thanks for reporting this issue @rmcnellis. A fix is on the way.

**Comment by clnsmth** [comment]
@rmcnellis This issue is now fixed in [version 3.5.3](https://github.com/EDIorg/EMLassemblyline/releases/tag/v3.5.3). Please let me know if you encounter any issues.
